Makefile

```diff
@@ -113,3 +114,14 @@ gendocs: docs/minikube.md
 docs/minikube.md: $(shell find cmd) $(shell find pkg/minikube/constants) pkg/minikube/cluster/assets.go
 	$(MKGOPATH)
 	cd $(GOPATH)/src/$(REPOPATH) && go run -ldflags="$(K8S_VERSION_LDFLAGS) $(MINIKUBE_LDFLAGS)" -tags gendocs gen_help_text.go
+
+.PHONY: deb
+deb: 
```

Comment:
This makes a lot of sense but it not clear to me how to do that in a Makefile, any advice would be appreciated thanks!